zerver/openapi/zulip.yaml

```diff
@@ -7189,11 +7201,17 @@ paths:
                         example: 1
                       delivery_email:
                         type: string
+                        nullable: true
                         description: |
-                          The user's real email address. This field is present only if
-                          [email address visibility](/help/restrict-visibility-of-email-addresses) is
-                          limited and you are an administrator with access to real email addresses
-                          under the configured policy.
+                          The user's real email address. This value will be `None` if you cannot
+                          access this user's real email address. For bots, this field is always
+                          set to the real email of the bot, since everyone can access the bot's
+                          email irrespective of `email_address_visibility` setting.
```

Comment:
I think this would be better expressed as "because bot users always have EMAIL_ADDRESS_VISIBILITY_EVERYONE`? I checked and bots can't change `.email_address_visibility` via the API, so that is an invariant, and I think it's simper to explain it as such here, even if we short-circuit this in various code paths with an `is_bot` check.